Xamarin.Forms.Platform.iOS/Extensions/FontExtensions.cs

```diff
@@ -38,7 +38,12 @@ static UIFont _ToNativeFont(string family, float size, FontAttributes attributes
 								traits = traits | UIFontDescriptorSymbolicTraits.Italic;
 
 							descriptor = descriptor.CreateWithTraits(traits);
+
+							if (descriptor != null)
+								throw new Exception();
```

Comment:
This PR, in case of null `UIFontDescriptor`, throws an exception handled by https://github.com/xamarin/Xamarin.Forms/blob/869868fa7b9483079f57b734e5be0a5a50fe2a36/Xamarin.Forms.Platform.iOS/Extensions/FontExtensions.cs#L72
After that, we use a default font setting the font size and the attributes.